boards/sim/sim/sim/configs/rpserver/defconfig

```diff
@@ -19,7 +19,7 @@ CONFIG_DEV_SIMPLE_ADDRENV=y
 CONFIG_EXAMPLES_RPMSGSOCKET=y
 CONFIG_FS_HOSTFS=y
 CONFIG_FS_PROCFS=y
-CONFIG_FS_RPMSGFS=y
+CONFIG_FS_RPMSGFS_SERVER=y
```

Comment:
Should we note such transition in release notes, so users will not get unexpected effect?

Reply:
Add "Breaking change" mark